src/lib/stickers.ts

```diff
@@ -213,7 +213,7 @@ export async function getEmojiForSticker(id: string, key: string, stickerId: num
  * set.
  */
 export function fuzzySearchStickerPacks(needle: string, haystack: Array<StickerPack>): Array<StickerPack> {
-  const searchKeys = ['manifest.title', 'manifest.author', 'meta.tags', 'meta.source'];
+  const searchKeys = ['manifest.title', 'manifest.author', 'meta.tags'];
```

Comment:
Fuzzy search started to break down with a lot of packs. In general, the main issue was any long string tended to have a lot of false positives in the fuzzy filter. I've mitigated this for now by:
* no longer filtering on source (it tended to only produce unexpected results)
* breaking the tags key up into an array, so each tag is searched individually rather than the full list of tags as one string

Reply:
This is a great! :+1: